docs/generateScript.md

```diff
@@ -0,0 +1,34 @@
+# Script Explanation: `generate.sh`
+## Prerequisites
+Before running the script, ensure the following tools are installed and properly configured on your system:
+- **kubectl**: Required to interact with the Kubernetes cluster and fetch the OpenAPI specification.
+- **pq**: A utility used for processing the fetched data.
+- **openapi-typescript**: A tool used to generate TypeScript types from the OpenAPI specification.
+
+## How to Use
+To run the `generate.sh` script, follow these steps:
+1. Navigate to the backend directory:
+    ```bash
+    cd backend
+    ```
+2. Execute the script using the following command:
+    ```bash
+    npm run generate:types
+    ```
+This will fetch the latest OpenAPI specification and generate the TypeScript types for the custom resources.
+## Purpose
```

Comment:
Maybe we should emphasize that here we just generate volcano related resources to develop codes.